### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by Ugboaja-Uchechi**
### Project Requirements

Add the styles of your choice to make the layout work, according to the wireframe.
Built with HTML, CSS, JAVASCRIPT,webpack

### CODE REVIEW

**Comment by Ugboaja-Uchechi** [comment]
All linter checks have passed.

File: README.md [hunk 1]
```diff
@@ -6,7 +6,8 @@
 
 # Screenshot
 
-![image](https://user-images.githubusercontent.com/74814780/136938704-30f68461-342b-44a4-a1a5-abadc4a6856d.png)
+![image](https://user-images.githubusercontent.com/74814780/137373744-4e73194f-c343-46ff-993f-7ac92ed88233.png)
```
Comment (helman101) [inline README.md:9]: - [x] Please use a screenshot of your project, recruiters and colleagues will want a preview of how the actual project looks 😄

**Review by helman101 — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 

Hi @Ugboaja-Uchechi 👋, 

Nice work so far on this milestone of the project  😄, but there are still some changes to do.

### Required Changes ♻️

- [ ] _Chack the comments below_

### Optional Changes 🤓 

- - [x] There are 2 issues in your project, that maybe were not spotted in the last milestone, kindly fix them if you want your project at its 100%:
    - First, your refresh button is not working.
    - Second, your scores appear only when i submit a score, it should not work this way.
    laderboard

Cheers and Happy coding!👏👏👏

gif

If anything is not 100% clear please leave your questions or comments in the PR thread, we will be glad to help you 😄.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Ugboaja-Uchechi) [reply README.md:9]: Hi, Thank you for the review. There is a screenshot of my project in the styling readme branch.

**Review by sinansevgi — APPROVED** [review]
# Approved :heavy_check_mark: 

Hi @Ugboaja-Uchechi , 

Your project is complete!  
Congratulations! 🎉

You can merge this pull request and continue with your journey :wave: 


Happy Hacking! :computer: 

-----

You can reply to this conversation if you have any questions or need elaboration. :wave: 
Sinan Sevgi


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**